bftengine/include/bftengine/Replica.hpp

```diff
@@ -35,6 +35,11 @@ enum MsgFlag : uint8_t {
   HAS_PRE_PROCESSED_FLAG = 0x4
 };
 
+class IControlHandler {
```

Comment:
This needs a comment. AFAIU, a control handler is responsible for infrastructure/system-level operations which are transparent to application handlers (the "execute" callback). In any case, we should have a comment that explicitly states the use-case/intend and maybe even expands on the notion of "Control State".

Reply:
done